api/tests/integration/infrastructure/repositories/organization-repository_test.js

```diff
@@ -2,7 +2,8 @@ const { catchErr, expect, knex, domainBuilder, databaseBuilder } = require('../.
 const { NotFoundError } = require('../../../../lib/domain/errors');
 const Organization = require('../../../../lib/domain/models/Organization');
 const organizationRepository = require('../../../../lib/infrastructure/repositories/organization-repository');
-const _ = require('lodash');
+const { ORGA_IDENTITY_PROVIDER } = require('../../../../lib/domain/constants').PIX_ORGA;
+const { times, map } = require('lodash');
```

Comment:
On fait ça côté front pour profiter du treeshaking mais est-ce que ça a un intérêt côté API ?

Aussi, je l'aurais mis dans un autre commit "refacto" car ce n'est pas lié à l'ajout de la propriété  au modèle.

Reply:
J'ai posé la question sur engineering, pas utile côté back mais je trouve ça cool d'avoir les même règles côté back et front, ça fait un seule règle à retenir.
Ok sur le principe de mettre les améliorations en SR mais là j'ai corrigé le test et c'est en le corrigeant que j'ai vu l'import lodash.
Je sais pas si ça apporte plus de clarté de corriger le test puis de faire avant la modification lodash (ou faire le commit sr avant).
What do you think ?
Pas de souci pour faire 2 commits séparés